hid/patches/platformio-stm32f1-no-serial-usb.patch

```diff
@@ -0,0 +1,11 @@
+--- aaa/tools/platformio-build-stm32f1.py	2022-07-16 18:54:42.536695468 +0200

```

Comment:
Wrong newlines in this file

Reply:
This was done on purpose. Destination file has CRLF so without it patch was not working.

Reply:
This is a a python code, why it's not working? Please show the error

Reply:
https://unix.stackexchange.com/questions/239364/how-to-fix-hunk-1-failed-at-1-different-line-endings-message

Hunk #1 FAILED at 1 (different line endings). <- otherwise I'm getting this kind of error

Reply:
AH this is a patch, I'm stupid. Okay.